packages/shared/src/hooks/useAccountEmailFlow.ts

```diff
@@ -48,12 +62,12 @@ function useAccountEmailFlow(
         action: emailFlow.ui.action,
         params: {
           email,
-          method: 'link',
+          method: 'code',
```

Comment:
We will need to make this come from the parameters. This is also used for resending the account verification email.

Reply:
Hmm does it use the same function?
I thought that was send automatically via Kratos?

Reply:
It sends automatically, but if you want to re-send the email (once the timer ends after registration, or in the profile->security page), we do it manually in this flow. Just not sure if this will work the same.

Reply:
Adjusted it, it's technically a different flow, but made sure both can live together.